src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -40,6 +54,76 @@ def config(**kwargs) -> dict:
     }
 
 
+@register(
+    "activations_stats",
+    "1.0",
+    format="csv",
+    description="Stats for activations",
+)
+def activation_stats(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    has_ended_at_states = [
+        ActivationStatus.FAILED.value,
+        ActivationStatus.STOPPED.value,
+        ActivationStatus.COMPLETED.value,
+        ActivationStatus.UNRESPONSIVE.value,
+        ActivationStatus.ERROR.value,
+        ActivationStatus.WORKERS_OFFLINE.value,
+    ]
+    counts_query = (
+        models.RulebookProcess.objects.filter(activation_id=OuterRef("pk"))
+        .values("activation_id")
+        .annotate(counts=Count("id"))
+        .values("counts")
+    )
+
+    activations = (
+        models.Activation.objects.annotate(
+            ended_at=Case(
+                When(status__in=has_ended_at_states, then=F("modified_at")),
+                output_field=DateTimeField(),
+            ),
+            counts=Coalesce(Subquery(counts_query), Value(0)),
+            max_restart_count=Value(
+                int(settings.ACTIVATION_MAX_RESTARTS_ON_FAILURE),
+                output_field=IntegerField(),
+            ),
+        )
+        .filter(
+            Q(created_at__gt=since, created_at__lte=until)
+            | Q(modified_at__gt=since, modified_at__lte=until)
+        )
+        .distinct()
+    ).values(
+        "id",
+        "name",
+        "status",
+        "restart_count",
+        "failure_count",
+        "log_level",
+        "organization_id",
+        "created_at",
+        "ended_at",
+        "counts",
```

Comment:
"counts" is not a bit misleading or generic name? Should it be renamed to "activation_instances_count" or something more explicit?

Reply:
This name came from the MVP user stories, to me I feel `activation_counts` is more straightforward.

Reply:
To me activations_counts seems good as well

Reply:
changed to `activations_counts`